server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/bucket/GeoDistanceIT.java

```diff
@@ -56,7 +56,7 @@ protected boolean forbidPrivateIndexSettings() {
         return false;
     }
 
-    private final IndexVersion version = IndexVersionUtils.randomCompatibleVersion(random());
+    private final IndexVersion version = IndexVersionUtils.randomCompatibleWriteVersion(random());
```

Comment:
All of the IT tests create real indices, hence they can't use read-only versions.